docs/mine-hnt/validators/setup.mdx

```diff
@@ -0,0 +1,162 @@
+---
+id: validator-run
+title: Run a Validator
+hide_title: true
+sidebar_label: Run a Validator
+slug: /mine-hnt/validators/validator-run
+---
+
+import useBaseUrl from "@docusaurus/useBaseUrl";
+
+# Run a Validator
+
+The following options exist to build a validator node:
+
+* From [docker image](#docker-image)
+* From [source](#source)
+
+Regardless of the option, the genesis block needs to be installed:
+
+```
+Download command goes here
```

Comment:
Seems like we're missing a command here?

Reply:
yes, updated...